lxd/storage/drivers/driver_ceph_utils.go

```diff
@@ -589,10 +589,11 @@ func (d *ceph) rbdListVolumeSnapshots(vol Volume) ([]string, error) {
 	return snapshots, nil
 }
 
-// copyWithSnapshots creates a non-sparse copy of a container including its snapshots.
+// copyVolumeDiff creates a non-sparse copy of a volume by exporting and importing the diff
```

Comment:
Doesn't export-diff send a sparse copy?

Reply:
I think the original comment wants to differentiate between doing a copy using Ceph internal mechanisms (causing a dependency relation to the source vol) and copying the volumes using the import-diff/export-diff functions without having this relation to the source. 

But there is `rbd export` which actually is non-spare and we can say that `rbd export-diff` is in fact sparse but without dependency relation to its source vol.

I'll change the comment, good catch.

Reply:
Fixed.